src-plugins/itkFilters/itkFiltersToolBox.cpp

```diff
@@ -251,12 +251,16 @@ itkFiltersToolBox::itkFiltersToolBox ( QWidget *parent ) : medAbstractSelectable
     d->thresholdFilterValue = new QDoubleSpinBox;
     d->thresholdFilterValue->setRange ( -10000, 10000 );
     d->thresholdFilterValue->setValue ( itkFiltersThresholdingProcess::defaultThreshold );
+    d->thresholdFilterValue->setObjectName("thresholdValue");
```

Comment:
can you update itkFiltersToolBox.h accordingly, as in https://github.com/Inria-Asclepios/music/pull/643 ?